kong/plugins/ip-restriction/schema.lua

```diff
@@ -8,8 +8,8 @@ return {
     { config = {
         type = "record",
         fields = {
-          { whitelist = { type = "array", elements = typedefs.cidr_v4, }, },
-          { blacklist = { type = "array", elements = typedefs.cidr_v4, }, },
+          { whitelist = { type = "array", elements = typedefs.cidr_v4_v6, }, },
+          { blacklist = { type = "array", elements = typedefs.cidr_v4_v6, }, },
```

Comment:
Here as I commented above, you could just use:
```
typedefs.ip_or_cidr
```

Reply:
we can break existing third party plugins using validate_ip_or_cidr (view comment above)